### ISSUE TITLE
[Request] Information or a write-up of your process.

### ISSUE THREAD

**Issue by syntholly** [Enygma2002/simsPlumbob#1]
I understand this was a long time ago, but I'd love a detailed - or not - write-up of how you managed to do this.
I'm really curious how you got to the position of sending offsets and RGB values to the device. Can you step-by-step me?

**Comment by Enygma2002** [comment]
Hi, @vSanjo.

Somehow I've missed your message at the time.

Just in case it may still be somehow relevant to you, long story short, what I remember doing is:

1. Used a USB sniffer application for Windows and kept it open while the Sims game was connected to the plumbob and sending it messages to switch colors.
2. I've noticed the pattern that the only last 3 numbers were changing and they were the RGB components of the end color.
3. I used a similar python project (https://github.com/nmilford/stormLauncher) that was controlling a toy rocket launcher in order to quickly reproduce the communication method via USB.
4. I also used the USB sniffer to get the device ID, which was needed in the communication procedure.

Put everything together and voila. It was really much easier than I anticipated.

Cheers!
